client/src/App.js

```diff
@@ -1,12 +1,18 @@
-import React from "react"
-import "./App.css"
+import React, { Component } from "react"
+import { BrowserRouter as Router, Route } from "react-router-dom"
 
-function App() {
-  return (
-    <div>
-      <p>hello</p>
-    </div>
-  )
+import HomePage from "./Components/HomePage"
+
+class App extends Component {
+  render() {
+    return (
```

Comment:
It should be a functional component because you don't have state or lifecycle methods here. 

**Note:**
Functional components are much easier to read and test because they are plain JavaScript functions without state or lifecycle-hooks and you end up with less code